third_party/2and3/flask/testing.pyi

```diff
@@ -4,17 +4,22 @@
 
 from click import BaseCommand
 from click.testing import CliRunner, Result
-from typing import Any, IO, Iterable, Mapping, Optional, Union
+from typing import Any, IO, Iterable, Mapping, Optional, Union, Tuple, overload
+from typing_extensions import Literal
 from werkzeug.test import Client
+from flask.wrappers import Response
 
 def make_test_environ_builder(app: Any, path: str = ..., base_url: Optional[Any] = ..., subdomain: Optional[Any] = ..., url_scheme: Optional[Any] = ..., *args: Any, **kwargs: Any): ...
 
-class FlaskClient(Client):
+class FlaskClient(Client[Response]):
     preserve_context: bool = ...
     environ_base: Any = ...
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     def session_transaction(self, *args: Any, **kwargs: Any) -> None: ...
-    def open(self, *args: Any, **kwargs: Any): ...
+    @overload
+    def open(self, *args: Any, as_tuple: Literal[True], **kwargs: Any) -> Tuple[Mapping[str, Any], Response]: ...
+    @overload
+    def open(self, *args: Any, as_tuple: Literal[False] = ..., **kwargs: Any) -> Response: ...
```

Comment:
Is `open` even necessary here? It seems to have the same signature as `Client.open`.

Reply:
Dropped.